pavement.py

```diff
@@ -491,8 +543,8 @@ def test_integration(options):
             sh('sleep 30')
             call_task('setup_data')
         sh(('python manage.py test %s'
-           ' --noinput -v 3 --liveserver=0.0.0.0:8000' % name))
-    except BuildFailure, e:
+            ' --noinput --liveserver=localhost:8000' % name))
```

Comment:
change localhost to 0.0.0.0 because we need to expose it to QGIS Server containers